.github/workflows/build.yml

```
@@ -111,6 +111,9 @@ jobs:
         run: make smelly
       - name: Check limited ABI symbols
         run: make check-limited-abi
+      - name: Check for Unsupported C Global Variables
```

```suggestion
      - name: Check for unsupported C global variables
```

Minor nit: Use the same sentence case as the other step names